File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/sys/dztype/pojo/DzType.java
package com.yss.ams.base.information.support.sys.dztype.pojo;


import com.yss.framework.api.common.co.BasePojo;

/**
 * 对账类型字典表实体类
 * @author yuankai 公共组件拆分 2017.5.31
 *
 */
public class DzType extends BasePojo {
	
	private static final long serialVersionUID = 1095788588950793961L;

	/**
	 * 对账类型父级节点代码
	 */
	private String C_DZ_CODE_P = "";

	/**
	 * 对账类型代码
	 */
	private String C_DZ_CODE = "";

	/**
	 * 对账类型名称
	 */
	private String C_DZ_NAME = "";

	public String getC_DZ_CODE_P() {
		return C_DZ_CODE_P;
	}

	public void setC_DZ_CODE_P(String cDZCODEP) {
		C_DZ_CODE_P = cDZCODEP;
	}

	public String getC_DZ_CODE() {
		return C_DZ_CODE;
	}

	public void setC_DZ_CODE(String cDZCODE) {
		C_DZ_CODE = cDZCODE;
	}

	public String getC_DZ_NAME() {
		return C_DZ_NAME;
	}

	public void setC_DZ_NAME(String cDZNAME) {
		C_DZ_NAME = cDZNAME;
	}

}

File: AMS_基础信息组件/code/client/source/YssInformation.Support/Sys/Dictionary/Pojo/DzType.cs
using System.Text.RegularExpressions;
using Newtonsoft.Json;
using FAST.Common.Service.Datastructure;
using FAST.Common.Service.Pojo.Base;

namespace YssInformation.Support.Sys.Dictionary.Pojo
{
    /// <summary>
    /// 
    ///  作用：划款指令类型字典
    ///  
    ///  作者：陈尤龙
    ///  
    ///  日期：2012-11-03
    ///  
    ///  版本：v1.0.0.4
    ///  
    /// </summary>
    [NodeDesc(ParentNode = "C_DZ_CODE_P", TreeNode = "C_DZ_CODE")]
    public class DzType : BasePojo
    {
        /// <summary>
        /// 划款指令代码
        /// </summary>
        private string c_DZ_CODE = "";

        /// <summary>
        /// 划款指令名称
        /// </summary>
        private string c_DZ_NAME = "";

        /// <summary>
        /// 划款类型父级代码
        /// </summary>
        private string c_DZ_CODE_P = "";

        /// <summary>
        /// 划款类型父级代码
        /// </summary>
        [JsonProperty(PropertyName = "c_DZ_CODE_P")]
        public string C_DZ_CODE_P
        {
            get { return c_DZ_CODE_P; }
            set { c_DZ_CODE_P = value; }
        }

        /// <summary>
        /// 属性: C_DZ_CODE 
        /// </summary>
        [JsonProperty(PropertyName = "c_DZ_CODE")]
        public string C_DZ_CODE
        {
            set { c_DZ_CODE = value; }
            get { return c_DZ_CODE; }
        }

        /// <summary>
        /// 属性: C_DZ_NAME 
        /// </summary>
        [JsonProperty(PropertyName = "c_DZ_NAME")]
        public string C_DZ_NAME
        {
            set { c_DZ_NAME = value; }
            get { return c_DZ_NAME; }
        }
    }
}
